model/repo_ref.go

```diff
@@ -150,34 +150,45 @@ func FindRepoRefByOwnerAndRepo(owner, repoName string) (*RepoRef, error) {
 func (r *RepoRef) AddPermissions(creator *user.DBUser) error {
 	rm := evergreen.GetEnvironment().RoleManager()
 
-	newScope := gimlet.Scope{
-		ID:          GetRepoScope(r.Id),
-		Resources:   []string{r.Id},
-		Name:        r.Id,
-		Type:        evergreen.ProjectResourceType,
-		ParentScope: evergreen.UnrestrictedProjectsScope,
+	adminScope := gimlet.Scope{
+		ID:        GetRepoAdminScope(r.Id),
+		Resources: []string{r.Id},
+		Name:      r.Id,
+		Type:      evergreen.ProjectResourceType,
 	}
-	if err := rm.AddScope(newScope); err != nil {
+	if err := rm.AddScope(adminScope); err != nil {
 		return errors.Wrapf(err, "error adding scope for repo project '%s'", r.Id)
 	}
 
+	// will be used to request permissions at the repo level
+	unrestrictedBranchesScope := gimlet.Scope{
```

Comment:
This isn't used yet, but it's relevant so I wanted to test it now. We should at some point provide a way for users to request permissions for every unrestricted project in the repo from mana (restricted projects need to be asked for specifically). This will be used for that.